lib/clients/authenticated.js

```diff
@@ -3,8 +3,14 @@ const { signRequest } = require('../../lib/request_signer');
 const PublicClient = require('./public.js');
 
 class AuthenticatedClient extends PublicClient {
-  constructor(key, b64secret, passphrase, apiURI) {
-    super('', apiURI);
+  constructor(
+    key,
+    b64secret,
+    passphrase,
+    apiURI = 'https://api.gdax.com',
+    productID = null
```

Comment:
Ahh that's a very good point! Gotcha.

In that spirit, should the (authed) constructor throw an error if a `productID` is not explicitly provided?